python/tvm/contrib/target/vitis_ai.py

```diff
@@ -132,12 +132,12 @@ def vitis_ai_compiler(ref):
         layers = xgraph.get_layers()
 
         # Get the output tensor names using XGraph and output Relay ids
-        out_tensor_names = []
+        out_tensor_names = [1] * len(output_relay_ids)
```

Comment:
It would be better to make the type consistent. For example, you could use `["unknown_name" for _ in range(output_relay_ids)]`.